Upcoming-iCal-Events.widget/main.coffee

```diff
@@ -96,33 +124,27 @@ update: (output, domEl) ->
             nameAndCalendar = lines[i].split('(')
 
             if nameAndCalendar.length < 2
-            	continue
+                continue
 
             name = nameAndCalendar[0].replace(bullet, '')
             calendar = 'No Calendar'
 
             if nameAndCalendar[1] != undefined
                 calendar = nameAndCalendar[1].replace(')','')
 
-            if IGNORE_CALENDER.includes(calendar)
-                continue
-
             if ( name.length > MAX_CHARACTERS )
                 name = name.substr(0, MAX_CHARACTERS) + "..."
 
-            date = ((lines[i+1].split("at"))[1])
-            if ( date == undefined )
-                date = "ALLDAY"
-            else
-                date = date.substr(0,9)
```

Comment:
This is not required as we are not getting the end dates by using `-eed` option.